common/src/main/java/org/apache/rocketmq/common/ControllerConfig.java

```diff
@@ -24,7 +24,9 @@ public class ControllerConfig {
 
     private String rocketmqHome = System.getProperty(MixAll.ROCKETMQ_HOME_PROPERTY, System.getenv(MixAll.ROCKETMQ_HOME_ENV));
     private String configStorePath = System.getProperty("user.home") + File.separator + "controller" + File.separator + "controller.properties";
-
+    public static final String DLEDGER_CONTROLLER = "DLedger";
```

Comment:
would be better to use an enum

Reply:
Using enum is indeed more elegant, but I think using string constants here is sufficient.

Reply:
ok